address_book.rb

```diff
@@ -1,13 +1,22 @@
-class AddressBook
-  def initialize(csv_path)
-
-  end
+require 'csv'
 
-  def print_people
+class AddressBook
+	def initialize(csv_path)
+  	#Read CSV File
+  	@people = CSV.read(csv_path)
```

Comment:
Burada CSV'i okuyup array olarak @people'in icine atmissin, yapman gereken CSV'i okuyup Person objeleri olusturup onlari @people'in icine atmak